lib/foreman/renderer.rb

```diff
@@ -99,7 +113,10 @@ def snippet(name, options = {})
         begin
           return unattended_render(template, nil, options[:variables] || {})
         rescue => exc
-          raise "The snippet '#{name}' threw an error: #{exc}"
+          # fixes the backtrace but ideally should raise better error message with Foreman rendering exception, same everywhere in this file
```

Comment:
Not sure that I understand this comment, does it mean it should use RenderingException, or provide different information?